src/parsers/chromemanifest.js

```diff
@@ -0,0 +1,71 @@
+import readline from 'readline';
+
+
+export default class ChromeManifestParser {
+
+  constructor(stream, filename) {
+    this.stream = stream;
+    this.filename = filename;
+    this.triples = null;
+  }
+
+  filterTriples({subject, predicate, object} = {}) {
+    return this.parse()
+      .then((triples) => {
+        return new Promise((resolve) => {
+          resolve(triples.filter((triple) => {
```

Comment:
I'm used to seeing `resolve([args])` but not a function inside it.

It took me a second to make sense of this only because I don't expect it. I know it's slightly more verbose but what about:

```js
var filteredTriples = triples.filter((triple) => {
  var searches = {subject, predicate, object};
  for (let key in searches) {
    let val = searches[key];
    if (typeof val !== undefined && val === triple[key]) {
      return true;
    }
  }
});

resolve(filteredTriples);
```